index.html

```diff
@@ -30,6 +32,8 @@
 
 </head>
 <body>
+    <button id="themeselector1">Basic Theme</button>
```

Comment:
The id on both of these buttons should be something like `themeSelectorBasic` so it reflects the theme name